docs/source/workflow_outputs.rst

```diff
@@ -325,6 +325,20 @@ Annual unmet hours are listed below.
 
 These numbers reflect the number of hours during the heating/cooling season when the conditioned space temperature deviates more than 0.2 deg-C (0.36 deg-F) from the heating/cooling setpoint.
 
+Annual Unmet Loads
+~~~~~~~~~~~~~~~~~~
+
+Annual unmet loads are listed below.
+
+  ========================================  =====
+  Type                                      Notes
+  ========================================  =====
+  Unmet Loads: Hot Water Shower Energy (J)  TODO
+  Unmet Loads: Hot Water Shower Time (hr)   TODO
+  ========================================  =====
+
+TODO
```

Comment:
What if we combine these with the above Annual Unmet Hours like so:

- `Unmet Hours: Showers (hr)`
- `Unmet Hours: Showers (%)`

Also document that this accounts for a storage water heater running out of water and occupant discomfort. It does not account for temperature drop due to a distribution system or a tankless water heater's minimum firing rate. Using smooth schedules (rather than detailed/stochastic schedules) may under-represent these values.

Reply:
What should we report for timeseries, just `Unmet Hours: Showers (hr)`?

Also, seems like if we're no longer reporting unmet loads in kBtu, then we'd no longer need to use shower peak flow in the EMS program? If not, this should simplify down to just comparing Use Side temperature to Mixed temperature during shower draws.

Reply:
Good point, yes the % output would only make sense as an annual output.

Second question is for @jmaguire1.

Reply:
Just the hours in the timeseries makes a lot of sense.

If we drop the energy metric, and just track hours of unmet loads, then we could drop the flow rate. The original intent with including flow rate and calculating energy, rather than just hours, was that we might want to account for difference between a shower that's just below setpoint and one that's like 30F below setpoint, or a small amount of flow vs. multiple showers at the same time being below setpoint. I've been proposing energy as the more important metric for a few years now, but the problem (as we discussed) is no one has any context of what 1 kWh of unmet showers means.

All that being said: I'm OK with dropping energy and trying to just report unmet hours of showering, in which case I think it's fine to drop max flow rate from the EMS program. There's no real standard way to report this, and there's advantages to hours in terms of being more intuitive.

Reply:
Sounds good @jmaguire1. I've gone ahead and pulled the peak flow stuff out of the EMS program.